pom.xml

```diff
@@ -464,6 +464,7 @@
               </manifestEntries>
             </archive>
             <additionalparam>${javadoc.additional.params}</additionalparam>
```

Comment:
if additionalparam was replaced with additionalOptions the line should be removed

Reply:
I stated the reason why I decided not to in the PR description:

> I left `additionalparam` for backward compatibility in case someone uses old plugin by overwriting a version.

If you feel like it's not a reason good enough I can remove it.

Reply:
Sorry, I've missed that. May be it would be good to add the comment to the respective line why it is there (additionalparam line) and which versions of plugin it is meant for (both lines) ?

Reply:
Good idea, added in b04d45732c8b17878b90884b3833a947b40d6d96